src/Routes.js

```diff
@@ -29,20 +29,22 @@ const InsightsRoute = ({ component: Component, rootClass, ...rest }) => {
     const root = document.getElementById('root');
     root.removeAttribute('class');
     root.classList.add(`page__${rootClass}`, 'pf-c-page__main');
-    root.classList.add(`page__${rootClass}`, 'pf-l-page__main');
     root.setAttribute('role', 'main');
 
-    return <Route { ...rest } component={ Component } />;
+    return (<Component { ...rest } />);
 };
 
+type Props = {
+    childProps: any
+};
 InsightsRoute.propTypes = {
     component: PropTypes.func,
     rootClass: PropTypes.string
 };
 
-function checkPaths(routes) {
+function checkPaths (routes) {
     return some(Object
-    .values(routes), route => matchPath(location.href, { path: `${document.baseURI}platform/advisor${route}` }));
+    .values(routes), route => matchPath(location.href, { path: `${document.baseURI}insights/advisor${route}` }));
```

Comment:
This should be just `insights` no `advisor`.

`.values(routes), route => matchPath(location.href, { path: `${document.baseURI}insights${route}` }));`